lib/templates.js

```diff
@@ -173,45 +177,38 @@ const createAPIResourceTemplate = (serviceName, serviceDefinition, appConfig, pa
     if (apiResources.length > 0) return apiResources;
 };
 
-/**
- * This is a template function to create Event Resource object for Event Resource Array.
- *
- * @param {string} eventName The name of the event.
- * @param {object} eventDefinition The definition of the event.
- * @returns {Object} An object for the Event Resource.
- */
-// TODO: wrong name srvDefinition, should be model
-const createEventResourceTemplate = (eventName, eventDefinition, appConfig, packageIds) => {
+const createEventResourceTemplate = (serviceName, eventDefinition, appConfig, packageIds) => {
     const ordExtensions = readORDExtensions(eventDefinition);
     return {
-        ordId: `${appConfig.ordNamespace}:eventResource:${eventName}:v1`,
+        ordId: `${appConfig.ordNamespace}:eventResource:${serviceName}:v1`,
         title:
-            ordExtensions.title ??
             eventDefinition["@title"] ??
             eventDefinition["@Common.Label"] ??
             `ODM ${appConfig.appName.replace(/[^a-zA-Z0-9]/g, "")} Events`,
-        shortDescription: ordExtensions.shortDescription ?? "Example ODM Event",
-        description: ordExtensions.description ??
+        shortDescription: "Example ODM Event",
+        description:
             eventDefinition['@description'] ?? eventDefinition['@Core.Description'] ??
             "CAP Event resource describing events / messages.",
-        version: ordExtensions.version ?? "1.0.0",
-        releaseStatus: ordExtensions.releaseStatus ?? "beta",
+        version: "1.0.0",
+        releaseStatus: "beta",
```

Comment:
Not sure if "beta" is a good default here. I would go with active / stable:

```suggestion
        releaseStatus: "active",
```